### ISSUE TITLE
Avatars for users don't consistently load

### ISSUE THREAD

**Issue by Lyude** [baedert/corebird#486]
On about half of my DMs the avatars for the people I'm talking with don't seem to ever load (this persists through restarts):
![screenshot from 2015-12-30 22-23-58](https://cloud.githubusercontent.com/assets/1527057/12060844/34d8da54-af46-11e5-9538-e5412d1f04ed.png)

**Comment by baedert** [comment]
That's a duplicate of #193. Since we have to save all DMs in a database, we also have to save the avatar URL and if that changes we can't get a new URL. For example, if you go to the profile of one of those people first, and then go to the DM page, the avatar should be there (*should*).

We could make a fallback in the avatar cache to first load the user info and then take the avatar URL from there but *sigh*.

**Comment by Lyude** [comment]
Changing this because as of late, I've started experiencing this same issue on the main timeline. Like you mentioned for the DM tab, opening their profile page sometimes gets their avatar back (but only after restarting corebird it seems)

**Comment by baedert** [comment]
This *should* be fixed now. We have to do a user_id->user_info->avatar_url->avatar detour... but yeah.
